pkg/template/apis/template/types.go

```diff
@@ -108,10 +108,24 @@ type TemplateInstanceSpec struct {
 // TemplateInstanceRequester holds the identity of an agent requesting a
 // template instantiation.
 type TemplateInstanceRequester struct {
```

Comment:
explained this to @bparees off line ... the net, we can't generate protobufs with DefaultInfo